### ISSUE TITLE
0.60.6 Discussion

### ISSUE THREAD

**Issue by kelset** [react-native-community/releases#130]
Conversation on this thread are limited to 0.60.5 release's major issues and cherry-pick requests from commits that are already on master.

An example of a good such request is a bug fix for a serious issue that has been merged into master but did not make the 0.60.5 cut.

In other words, if you cannot point to a particular commit on master, then your request likely belongs as a new issue in http://github.com/facebook/react-native/issues.

If the commit you request to cherry pick is a complicated port, you will be asked to create the PR to the 0.60-stable branch yourself, in order to ensure that the process proceeds smoothly.

**Comment by maheshaarkenea** [comment]
"react-native eject" command not working after this upgrade. i deleted ios and android folder. now i have no option to create those again.

**Comment by kesha-antonov** [comment]
Good job guys

**Comment by Frekansapp** [comment]
I think autolinking causing problems with react-native-navigation see [here](https://github.com/react-native-community/cli/issues/467)

**Comment by gleydson** [comment]
I had the same problem @Frekansapp!

**Comment by Frekansapp** [comment]
@gleydson did you solve it?

**Comment by Ilphrin** [comment]
Thank you so much for your awesome work! =D Can't wait to have enough time to contribute myself fur future releases!
I don't see any commit that should be merged in a hurry for next minor release (even though I'd love to see the new jsc-android update and this ScrollView sticky header ;) )

**Comment by ImBeCiliC** [comment]
Hello this was also posted to 0.59.x and sadly didn't made it into the 0.59.10 release. Can the new Keyboard behaviour on Android come into the next 0.60.x release?
facebook/react-native@d88e470
It's from this old and long discussion facebook/react-native#14045
Thanks

**Comment by sunnylqm** [comment]
Please cherry pick https://github.com/facebook/react-native/commit/76c50c1db15a1040b185770072d960652e7974a3

**Comment by matamicen** [comment]
Thank you guys for the effort! 

CocoaPod is included in this version, so any ideas how is the new procedure to install for example react-native-firebase for admob? in release 0.58 We had to follow this [guide](https://dev-yakuza.github.io/en/react-native/react-native-firebase-admob/) but now with cocoapod included  is any other easier way to install it ? any help how to do it ?

Thanks!

**Comment by thymikee** [comment]
Bump CLI to ^2.2.0 https://github.com/facebook/react-native/commit/5b448eca4fe4f3a769a5b1359707ed1d40ad61e6

**Comment by IljaDaderko** [comment]
@matamicen most packages will have something called a `.podspec` file, so to add them through CocoaPods, you just need to point to dir where that `.podspec` file lives. Here is example with `react-native-firebase` v5

```ruby
# Firebase (note you need to include firebase pods as well for features that you use)
  pod 'Firebase', '~> 5.20.0'
  pod 'Firebase/Core', '~> 5.20.0'
  pod 'Firebase/Analytics', '~> 5.20.0'
  pod 'Firebase/DynamicLinks', '~> 5.20.0'
  pod 'Firebase/Messaging', '~> 5.20.0'

  # React-Native libraries (react-native-firebase)
  pod 'RNFirebase', :path => '../node_modules/react-native-firebase/iOS'
```

From here you just run `pod update` in your `ios` dir and it will install everything.

**Comment by dulmandakh** [comment]
prettier fix - https://github.com/facebook/react-native/commit/79853d6e53337c82f064aa80dd8b8ac8916afa9d

**Comment by sunnylqm** [comment]
@dulmandakh Do you mean this one? https://github.com/facebook/react-native/commit/7254bab0b3fa129cd238783ab993fbae1102d60a

**Comment by AndrewJack** [comment]
Can this iOS 13 fix get cherry picked? 
https://github.com/facebook/react-native/commit/46bdb4161c84b33f1d0612e9c7cdd824462a31fd

Related issue: https://github.com/facebook/react-native/issues/25181

**Comment by matamicen** [comment]
> @matamicen most packages will have something called a `.podspec` file, so to add them through CocoaPods, you just need to point to dir where that `.podspec` file lives. Here is example with `react-native-firebase` v5
> 
> ```ruby
> # Firebase (note you need to include firebase pods as well for features that you use)
>   pod 'Firebase', '~> 5.20.0'
>   pod 'Firebase/Core', '~> 5.20.0'
>   pod 'Firebase/Analytics', '~> 5.20.0'
>   pod 'Firebase/DynamicLinks', '~> 5.20.0'
>   pod 'Firebase/Messaging', '~> 5.20.0'
> 
>   # React-Native libraries (react-native-firebase)
>   pod 'RNFirebase', :path => '../node_modules/react-native-firebase/iOS'
> ```
> 
> From here you just run `pod update` in your `ios` dir and it will install everything.

@IljaDaderko thanks very clear!

**Comment by kelset** [comment]
quick update: I changed this conversation to about 0.60.3 aiming for 0.60.4 since [Hermes was micdropped during ChainReact](https://facebook.github.io/react-native/docs/hermes/) 😂

We'll likely do a new release soon-ish, as soon as [this fix for the CLI](https://github.com/facebook/react-native/pull/25478#issuecomment-511452946) is implemented so that we can cherry pick this https://github.com/facebook/react-native/commit/7254bab0b3fa129cd238783ab993fbae1102d60a

Also, because of Hermes landing in branch directly quite a few of the CI jobs are now failing, which lowers a bit my confidence on ensuring that the releases are as bug free as we can get them to be - so it's likely that patch releases will be small and limited while aiming for a quick rollout of 0.61 (more details about it will arrive in the next couple weeks, we are waiting for some things to land).

**Comment by fungilation** [comment]
To be clear of what you are saying. For CI fixes and larger changes in RN needed to accommodate Hermes, those are going to land in 0.61 instead? So it's reasonable that production apps should wait for 0.61 before using Hermes?

**Comment by kelset** [comment]
Oh no - sorry I meant that it broke some CI jobs that are set up in the 0.60 branch.

To clarify: you can use Hermes via 0.60.3 even in production (AFAIK - I haven't seen any major issues in the main repo), but simply since the CI is kinda broken in the 0.60-branch it means we have less ways to find eventual issues that may be caused by some cherry picks.

But the CI on master branch is actually been improved a lot since 0.60-branch was cut, and it's green again from what I can see.

**Comment by janicduplessis** [comment]
https://github.com/facebook/react-native/commit/88e18b6c8d562a4571cc48c202dda944cacfd742 to fix blob memory leak on Android. Might also need https://github.com/janicduplessis/react-native/commit/6c2f4f52feefc71a15fe7df4731440da9bdfa5b7 (hermes has not landed on master so can't land this diff) which caused a crash for me when trying out hermes.

**Comment by kelset** [comment]
@janicduplessis I think that that diff will have to be done as PR against the 0.60-branch to see it happen in 0.60.x because Hermes was landed first in the branch so I don't think that the code will be precisely the same once it's on master (so yeah, I fear you may having to end up doing that PR twice 😓)

**Comment by Ivan-Stashak-CardinalPeak** [comment]
I'm upgrading a project to RN 0.60.3 . Unfortunately, I'm getting the following error when I attempt to run on Android:

`WebView has been removed from React Native. It can now be installed and imported from 'react-native-webview' instead of 'react-native'. See https://github.com/react-native-community/react-native-webview`

I have run 'npm install --save react-native-webview' and confirmed that it is appropriately linked into the my underlying Android project (i.e. settings.gradle, build.gradle, imports, etc...). I've also made the appropriate changes to ensure I import WebView like so:

`import { WebView } from 'react-native-webview';
`
and ensure that I've removed all occurrences of:

`import { WebView } from 'react-native;`

It compiles fine, but I get the run-time error above.

Any thoughts?

**Comment by moonstruck** [comment]
@Ivan-Stashak-CardinalPeak Please report the issue to https://github.com/react-native-community/react-native-webview or React Native repo. Purpose of this thread is not issue reporting.

> Conversation on this thread are limited to 0.60.3 release's major issues and cherry-pick requests from commits that are already on master.

**Comment by Kennytian** [comment]
@Ivan-Stashak-CardinalPeak  see https://github.com/react-native-community/react-native-webview/blob/master/docs/Getting-Started.md

**Comment by mjmasn** [comment]
https://github.com/facebook/react-native/commit/b432b8f13b4871dcafd690e57d37298662712b50 please @kelset!

**Comment by matpaul** [comment]
svg animation with nativeDriver
https://github.com/facebook/react-native/commit/bdc530b9bbf6099563d64fb573c4e6cbc93f9c5d

**Comment by thymikee** [comment]
CLI bump: https://github.com/facebook/react-native/commit/fafe5ee0726061e3590b91d3b5cff04e33781f87

**Comment by artdent** [comment]
facebook/react-native@6ef9106 would be a useful cherrypick - it fixes `use_native_modules!` in certain projects that use Cocoapods. It's a fix spun out of facebook/react-native#25349, which is a broader umbrella issue.

**Comment by janicduplessis** [comment]
@kelset Actually the second commit doesn't depend on hermes landing on master so I created a PR for it https://github.com/facebook/react-native/pull/25720, when it lands we can cherry pick both commits.

**Comment by kelset** [comment]
Ok, thanks for the clarification @janicduplessis - I've subscribed to the new PR (thanks for doing it!) so I should be aware of when it lands.

@artdent as I have commented on the issue, that fix will only land on 0.61 - the commit you are referring to is only part 1 and it will involve breaking changes (as far as I am aware of).

Aside from all of this, couple of extra things:
* because of this https://github.com/facebook/react-native/issues/25622 basically we need to ensure that 0.60 uses 0.54.x of Metro (no `^` in the deps)
* @cpojer did a new 0.60.4 release with some Hermes related fixes sooo yeah, let's aim to cherry pick for 0.60.5

**Comment by artdent** [comment]
@kelset The breaking changes are in part 2. Part 1 doesn't attempt to fix `use_frameworks!`; it just fixes the missing header file `<double-conversion/double-conversion.h>` as reported in facebook/react-native#25349. I've just verified locally that cherry-picking facebook/react-native@6ef9106 into RN 0.60.3 fixes the missing header file in my project without introducing any other problems.

**Comment by thymikee** [comment]
@kelset the `0.x` is a special case in semver indicating unstable APIs and `^0.54.1` will resolve to `0.54.x`. See: https://jubianchi.github.io/semver-check/#/^0.54.1/0.55.0. It seems like folks there have issues with their package manager (npm, why I'm not surprised)

**Comment by kelset** [comment]
@artdent oh ok awesome, thanks for the clarification!

@thymikee yeah what I meant up there with 0.54.x was simply "the latest version of 54 of which I can't remember the precise name" ;D

**Comment by cihati** [comment]
@kelset is there an ETA for the release?

**Comment by OzzieOrca** [comment]
https://github.com/facebook/react-native/commit/14b455f69a30d128db384749347f41b03b9a6000 - Set rounded rectangle mask on TouchableNativeFeedback's ripples

**Comment by solonifer** [comment]
error: bundling failed: Error: Unable to resolve module `warnOnce` from `/Users/user/Documents/myapp/HB/node_modules/react-native/Libraries/react-native/react-native-implementation.js`: Module `warnOnce` does not exist in the Haste module map

react-native 0.60.4

tryied everything from https://github.com/facebook/react-native/issues/23943 with no luck :(

**Comment by HarshitMadhav** [comment]
@kelset Please can you cherry pick this one issue https://github.com/facebook/react-native/issues/21801 ?

**Comment by khat33b** [comment]
When building a release APK in Android, I am getting error while bundling the JS. This is not happening while running a debug app.

```
Loading dependency graph, done.
info Writing bundle output to:, /home/react-native/test/android/app/build/generated/assets/react/release/index.android.bundle
info Writing sourcemap output to:, /home/react-native/test/android/app/build/intermediates/sourcemaps/react/release/index.android.bundle.packager.map
info Done writing bundle output
info Done writing sourcemap output
info Copying 64 asset files
info Done copying assets
```
Then many errors are thrown which say that some variables in the `android/app/build/generated/assets/react/release/index.android.bundle` are not declared.

```
Stack dump:
0.      Program arguments: ../../node_modules/hermesvm/linux64-bin/hermes -emit-binary -out /home/tivintis/react-native/contv/android/app/build/generated/assets/react/release/index.android.bundle /home/tivintis/react-native/contv/android/app/build/generated/assets/react/release/index.android.bundle -O -output-source-map 
1.      Program arguments: ../../node_modules/hermesvm/linux64-bin/hermes -emit-binary -out /home/tivintis/react-native/contv/android/app/build/generated/assets/react/release/index.android.bundle /home/tivintis/react-native/contv/android/app/build/generated/assets/react/release/index.android.bundle -O -output-source-map 

> Task :app:bundleReleaseJsAndAssets FAILED

FAILURE: Build failed with an exception.

* Where:
Script '/home/react-native/test/node_modules/react-native/react.gradle' line: 142

* What went wrong:
Execution failed for task ':app:bundleReleaseJsAndAssets'.
> Process 'command '../../node_modules/hermesvm/linux64-bin/hermes'' finished with non-zero exit value 135
```

**Comment by mikehardy** [comment]
@khat33b - 
```
Conversation on this thread are limited to 0.60.4 release's major issues and cherry-pick requests from commits that are already on master.

An example of a good such request is a bug fix for a serious issue that has been merged into master but did not make the 0.60.4 cut.

In other words, if you cannot point to a particular commit on master, then your request likely belongs as a new issue in http://github.com/facebook/react-native/issues.
```

**Comment by deorukhkarchaitanya** [comment]
https://github.com/facebook/react-native/commit/c5ea18f7389fe821e7a9882e4b1b30b0a1b266f4

Fixes build failures for apps with App extension on iOS. Thanks!

**Comment by mguay22** [comment]
Same as @solonifer - unable to use Jest right now.

@cpojer Are you able to provide the commit that fixes the issue you previously mentioned here on
facebook/react-native#23943

> This was fixed on master (we stopped using haste) and it will be part of 0.61. If you are using a previous version of RN, make sure you are using `"jest": { "preset": "react-native" }` in your setup properly, otherwise it will not work.

**Comment by velvenkat** [comment]
@kelset Execution failed for task ':app:bundleReleaseJsAndAssets'.
> A problem occurred starting process 'command '..\..\node_modules\hermesvm\win64-bin\hermes''
error is getting while set enableHermes=true in build.gradle file

**Comment by kdawgwilk** [comment]
Seems like this should be renamed to `0.60.5` now that `0.60.4`? I would love to see facebook/react-native#25619 cherry picked along with facebook/react-native#25818 & facebook/react-native#25816 but I fear that won't happen because there are possibly breaking changes with header imports for 3rd party libraries.  😢  `0.60.x` switch to CocoaPods broke our CocoaPods setup we already were using due to those issues we are stuck on `0.59.x` for now

**Comment by diegolmello** [comment]
@kelset Can you cherry pick https://github.com/facebook/react-native/commit/c5ea18f7389fe821e7a9882e4b1b30b0a1b266f4?
Thank you!

**Comment by thymikee** [comment]
Fixes template build failed in release mode: https://github.com/facebook/react-native/commit/0fcaca8e26223e47cc101c76bb7a7b02e0fc7101

**Comment by JuanJo4** [comment]
Adding the debug.keystore file back https://github.com/facebook/react-native/commit/d55025694be8b4ee5d09c8fdc910d42a5f144883

**Comment by janicduplessis** [comment]
Merged https://github.com/facebook/react-native/commit/ffdf3f22c68583fe77517f78dd97bd2e97ff1b9e. We had to make a new PR instead of cherry-picking since the hermes commit on 0.60 and master is slightly different.

**Comment by HarshitMadhav** [comment]
@janicduplessis Hi! this comment on the issue https://github.com/facebook/react-native/issues/21801#issuecomment-460833140 leading to a comment of yours regarding regression https://github.com/facebook/react-native/issues/15445#issuecomment-326680200 I want to know whether this small issue has been fixed?

**Comment by spyshower** [comment]
Not sure if this is the right place to ask but, do we have an estimate for the `0.61` release?

**Comment by grabbou** [comment]
@spyshower there's no estimation, but we are planning to ship it in the next two weeks. Updates will follow.

I am also back to the releases train and will be taking over the work on 0.60.5 as started by @kelset. It's been a while and I've been enjoying my OSS break, but back to duty.

I am going to cherry-pick the commits in this thread later today & tomorrow an update. I am hoping it will go through without major issues.

**Comment by jinshin1013** [comment]
[iOS] Fixes renderingMode not applied to GIF images: https://github.com/facebook/react-native/pull/24794#issue-277680176 

Thanks!!

**Comment by grabbou** [comment]
@jinshin1013 I see this commit is already part of 0.60.4. Have you tried running your app with that version?

**Comment by grabbou** [comment]
I just ended up going through this thread and looking at the cherry-pick requests. I wanted to thank everyone for participation and discussing the changes.

Here's the list of the commits that I am going to cherry-pick:
facebook/react-native@d88e470
facebook/react-native@76c50c1
facebook/react-native@5b448ec
facebook/react-native@a162554
facebook/react-native@7254bab
facebook/react-native@46bdb41
facebook/react-native@b432b8f
facebook/react-native@fafe5ee
facebook/react-native@fafe5ee (thanks for clarification later in the thread)
facebook/react-native@14b455f
facebook/react-native@c5ea18f

Here's some that need clarification:
facebook/react-native@d550256  @janicduplessis: facebook/react-native@88e18b6 - what’s the status of your PR?

Here's some of the commits that we are not going to cherry-pick:
@matpaul: facebook/react-native@bdc530b - this is pretty heavy diff and seems like a new feature too. I’d be more confident shipping that as a part of next release
@HarshitMadhav: as per the main thread comment, you need to point to a merged PR instead of an issue/open PR.
@kdawgwilk: yes. CocoaPods support to land in 0.61.

**Comment by grabbou** [comment]
I am going to cherry-pick them tomorrow and attempt releasing a new version. Meanwhile, I have also updated the title of this issue to reflect the new version.

**Comment by sunnylqm** [comment]
Can I take the last train? https://github.com/facebook/react-native/commit/c21e36db45167e443aba5a6bf4a3732e4989b306

**Comment by HarshitMadhav** [comment]
@grabbou Here is a commit about the issue I was suggesting to be cherry-picked. It's a very old commit https://github.com/facebook/react-native/commit/f954f3d9b674b13977f722bc3b8dc6c1b99fe6c7

**Comment by grabbou** [comment]
Didn't cherry-pick: 
- facebook/react-native@14b455f due to conflicts (depends on other commits that can't be merged clearly)
- facebook/react-native@f954f3d as it's already part of 0.60.4

Waiting on confirmation re: Hermes from Facebook and we are good to go. I will resume working on it tomorrow, I hope to publish it by then.

**Comment by grabbou** [comment]
You can subscribe to tests here: https://circleci.com/workflow-run/88733632-cb8c-47ad-aba1-620a1dd5bef5 

We will see what passes and what not and whether that's something that will stop us from making the release (related failure).

**Comment by HarshitMadhav** [comment]
@grabbou Thanks! Eagerly waiting for the new release with new fixes.

**Comment by grabbou** [comment]
The https://github.com/facebook/react-native/commit/c21e36db45167e443aba5a6bf4a3732e4989b306 didn't merge clearly because it's based on https://github.com/facebook/react-native/commit/d7f5153cd8d164fda9008c1cda51278a2dcbec9c (Hermes support added to master), instead of https://github.com/facebook/react-native/commit/e857d7066be504a9e90042be8ff919d62939895f (Hermes support added directly to this PR).

These commits both add Hermes support, but have tiny differences in Gradle files in terms of how do they resolve AAR files. Likely not relevant here, but just letting everyone know of my discovery.

**Comment by Johan-dutoit** [comment]
Not sure if I'm too late, but any chance we can get this merged in https://github.com/facebook/react-native/commit/8131b7bb7b4794e0e7003a6e3d34e1ebe4b8b9bc?

**Comment by jeffreyffs** [comment]
@Johan-dutoit those Cocoapod updates have already been rejected from a patch release (see https://github.com/react-native-community/releases/issues/130#issuecomment-518392397 and https://github.com/react-native-community/releases/issues/130#issuecomment-516598766 from above). Those changes will be landing in 0.61, which we will hopefully see soon.

**Comment by Johan-dutoit** [comment]
@jeffreyffs Bah! I searched the commits id's and found nothing.  Thanks!

**Comment by Justkant** [comment]
@grabbou Maybe a bit late for this release, but can this commit be cherry picked as well https://github.com/facebook/react-native/commit/d2b92fffb1d14dd0ec628e9dcdfd76e39f2067ff ?

This commit could help people experiencing issues with some prettier rules, but it can also wait :v:

**Comment by gaodeng** [comment]
Any chance we can get this cherry picked https://github.com/facebook/react-native/commit/6764385c8062fc647d016028351d8d7d28387ed5 
It fix the hermes issue https://github.com/facebook/hermes/issues/65 here

**Comment by grabbou** [comment]
@Johan-dutoit, @Justkant, and @gaodeng - thank you for your submissions, but we need to stop cherry-picking at some point and do testing.

This release is already quite a bit, I'd rather cut the next release soon or another minor version.

I am working on the CircleCI right now and plan to release it today. Just to give you an idea - the CI is failing and I need to check whether failures are the same as for past versions and in that case - we are good to go.

**Comment by raajnadar** [comment]
Can we get this cherry picked in next release?? **[Fix some languages wrapped texts are cut off on android]** https://github.com/facebook/react-native/commit/76c50c1db15a1040b185770072d960652e7974a3

**Comment by kelset** [comment]
As Mike said above

>we need to stop cherry-picking at some point and do testing.

Also, Mike is on holiday this week so I'll try this afternoon to see if 0.60 branch is ready for the 0.60.5 release (so yeah, testing and then eventually pushing & releasing).

**Comment by raajnadar** [comment]
I read that comment I should have mentioned v0.60.6! That's why I said in next release I thought 0.60.5 is already released.

**Comment by kelset** [comment]
quick update: I've tested locally the branch, and since everything seems ✅ I've published 0.60.5.

Changelog/release notes coming soon, it's in review (https://github.com/react-native-community/releases/pull/137).

In terms of 0.60.6, I don't think we will do it: as mentioned a while ago, having Hermes in the branch made the CI unstable so our confidence in making new releases is lower. So the plan is to instead release the first RC for 0.61 soon-ish (sometime over the next couple weeks).

(of course, this is under the assumption that 0.60.5 didn't introduce any major un-catched bugs... which it shouldn't 🤞)

**Comment by sayem314** [comment]
Fishhook `.podspec` file is missing on 0.60.5

**Comment by hackdie** [comment]
@sayem314 you don't need it anymore, it's on the changeLog

**Comment by sayem314** [comment]
> @sayem314 you don't need it anymore, it's on the changeLog

But `pod install` is failing with this error message `no podspec found for .......react-native/...fishhook` or something. I have deleted `yarn.lock, node_modules, Pods, and Podfile.lock` still the same. So I have downgraded back to 0.60.4 and everything is fine now.

**Comment by kevinwu127** [comment]
this happened to me as well. I needed to downgrade for `pod install` to work. What's going on?

**Comment by owinter86** [comment]
@sayem314 and @kevinwu127 you need to remove `pod 'React-fishhook', :path => '../node_modules/react-native/Libraries/fishhook'` from your podfile as its no longer needed.

You can also use https://react-native-community.github.io/upgrade-helper/ to see what files have been changed in upgrade.

**Comment by sayem314** [comment]
Oh, I see. Thanks, I would do that tomorrow!

**Comment by mikehardy** [comment]
Indeed @owinter86 - all hail the might upgrade-helper - note that react-native follows no semver convention and breaking changes show up in patches. You can *never* safely upgrade without checking the diff from upgrade-helper (or rn-diff-purge and before that rn-diff, same team as upgrade-helper, the tool just keeps evolving and upgrade-helper is the current best). Luckily this one is easy

**Comment by sijad** [comment]
can we have https://github.com/facebook/react-native/commit/c21e36db45167e443aba5a6bf4a3732e4989b306 for next release? I'm not sure why it did not cherry-picked for `0.60.5`

**Comment by benevbright** [comment]
Hi, guys. Thanks for addressing Android ScrollView vibrating issue with `TranslateY`.
I just tried the latest RN, 0.60.5. But the issue is still happening. 

Feel free to try this example. 
https://github.com/benevbright/react-navigation-collapsible/tree/master/example

#### GIF
![Aug-18-2019 6-19-48 PM](https://user-images.githubusercontent.com/916690/63230353-cd853a00-c20b-11e9-9fec-f1f0674a5c00.gif)

**Comment by HarshitMadhav** [comment]
@benevbright I tried with 0.60.4 version this issue still persists.

**Comment by kelset** [comment]
@sijad please read the explanation [here](https://github.com/react-native-community/releases/issues/130#issuecomment-519526533).

> react-native follows no semver convention and breaking changes show up in patches.

Yeah, as harsh as it may sound semver is not really followed - but we strive to, for patch (x.x.1) releases to not have changes, the fishhook one was a mistake, sorry 😓

@benevbright @HarshitMadhav please open an issue in the main repo for it.

--

General note: we will likely *not* do a next 0.60.x release because of what said previously about faulty CI. We are trying to get a first 61 RC up and running soon.

**Comment by benevbright** [comment]
@kelset 

We have open issue in the main repo. It's lasted for a long time.
https://github.com/facebook/react-native/issues/21801

I mentioned about it here, because https://github.com/react-native-community/releases/issues/130#issuecomment-519191161 this comment seems to have been trying fix that, but it's actually not resolved.

**Comment by sunnylqm** [comment]
@benevbright Seems a misunderstanding from the beginning.  https://github.com/facebook/react-native/issues/21801#issuecomment-522488647

**Comment by benevbright** [comment]
@sunnylqm You're so true. 😖 Thanks. 🙏

**Comment by HarshitMadhav** [comment]
@sunnylqm You are right! 😓

**Comment by grabbou** [comment]
Thanks @kelset for all the help! I had an accident that week and couldn't finish the release myself! Now, I am working on the 0.61 in a different thread.

**Comment by blohamen** [comment]
Hi all. Can you please make attention on this issue. Because now it's impossible to use selection and onSelectionChange on Android. https://github.com/facebook/react-native/issues/26047

**Comment by kristoff2016** [comment]
Hello there, 
Currently, I have the issue the same as this https://github.com/facebook/react-native/issues/25771
I hope it got fix it soon

Thanks

**Comment by grabbou** [comment]
Will be soon releasing another RC with this hot-fix https://github.com/facebook/react-native/pull/26462 for React Native Windows builds.

**Comment by janicduplessis** [comment]
I finally managed to land the Android blob fix properly. Hopefully it can merge cleanly on 0.60, if not I guess we'll just have to wait for 0.61

https://github.com/facebook/react-native/commit/9b2374b542f87b7baefcfb4a3eb4f57029069b57

**Comment by sunnylqm** [comment]
Since xcode 11 released before 0.61 stable, this fix seems necessary for 0.60 https://github.com/facebook/react-native/commit/46c7ada535f8d87f325ccbd96c24993dd522165d

**Comment by grabbou** [comment]
I am closing this issue as we are shipping 0.61.0 release right now. Please give it a try and comment on the appropriate issue thread if necessary.

**Comment by agrawalsurabhi89** [comment]
Redefinition of Module React , previously defined here in Dev Support module, Any idea. I have upgraded react-native version from 0.59.8 to 0.60.6. Any idea anyone?

**Comment by HarshitMadhav** [comment]
Hi @agrawalsurabhi89  You have posted the issue in a wrong discussion thread. Please consider raising a new issue or look out for the already open issues in the https://github.com/facebook/react-native/issues
